src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+
+const isObject = (value) => typeof value === 'object' && value !== null;
+
+const getAverageAge = people => isObject(people[0])
```

Comment:
In the second `reduce`, you do not return the average value, but only calculate the sum, so the name `getAverageAge` does not fit here and it can be transferred to a separate function.